src/modules/string/index.ts

```diff
@@ -0,0 +1,425 @@
+   *
+   * @param options Either the number of characters or an options instance. Defaults to `{ length: 1, casing: 'lower', bannedChars: [] }`.
+   * @param options.length The number of characters to generate. Defaults to `1`.
+   * @param options.casing The casing of the characters. Defaults to `'lower'`.
+   * @param options.bannedChars An array with characters to exclude. Defaults to `[]`.
+   *
+   * @example
+   * faker.string.alpha() // 'b'
+   * faker.string.alpha(10) // 'qccrabobaf'
+   * faker.string.alpha({ length: 5, casing: 'upper', bannedChars: ['A'] }) // 'DTCIC'
+   *
+   * @since 8.0.0
+   */
+  alpha(
+    options:
+      | number
+      | {
+          length?: number;
+          casing?: Casing;
+          bannedChars?: readonly LiteralUnion<AlphaChar>[] | string;
+        } = {}
+  ): string {
+    if (typeof options === 'number') {
+      options = {
+        length: options,
+      };
+    }
+
+    const { length = 1, casing = 'mixed' } = options;
+    let { bannedChars = [] } = options;
+
+    if (typeof bannedChars === 'string') {
+      bannedChars = bannedChars.split('');
+    }
+
+    if (length <= 0) {
+      return '';
+    }
+
+    let charsArray: string[];
+    switch (casing) {
+      case 'upper':
+        charsArray = [...UPPER_CHARS];
+        break;
+      case 'lower':
+        charsArray = [...LOWER_CHARS];
+        break;
+      case 'mixed':
+      default:
+        charsArray = [...LOWER_CHARS, ...UPPER_CHARS];
+        break;
+    }
+
+    charsArray = charsArray.filter((elem) => !bannedChars.includes(elem));
+
+    if (charsArray.length === 0) {
+      throw new FakerError(
+        'Unable to generate string, because all possible characters are banned.'
+      );
+    }
+
+    return Array.from({ length }, () =>
+      this.faker.helpers.arrayElement(charsArray)
+    ).join('');
+  }
+
+  /**
+   * Generating a string consisting of alpha characters and digits.
+   *
+   * @param options Either the number of characters or an options instance. Defaults to `{ length: 1, casing: 'mixed', bannedChars: [] }`.
+   * @param options.length The number of characters and digits to generate. Defaults to `1`.
+   * @param options.casing The casing of the characters. Defaults to `'mixed'`.
+   * @param options.bannedChars An array of characters and digits which should be banned in the generated string. Defaults to `[]`.
+   *
+   * @example
+   * faker.string.alphanumeric() // '2'
+   * faker.string.alphanumeric(5) // '3e5v7'
+   * faker.string.alphanumeric(5, { bannedChars: ["a"] }) // 'xszlm'
+   *
+   * @since 8.0.0
+   */
+  alphanumeric(
+    options:
+      | number
+      | {
+          length?: number;
+          casing?: Casing;
+          bannedChars?: readonly LiteralUnion<AlphaNumericChar>[] | string;
+        } = {}
+  ): string {
+    if (typeof options === 'number') {
+      options = {
+        length: options,
+      };
+    }
+
+    const { casing = 'mixed', length = 1 } = options;
```

Comment:
Might be nit picky:

```suggestion
    const { length = 1, casing = 'mixed' } = options;
```

For consistency with the signature + other methods.

Alternatively, sort the deconstructors alphabetically, if possible.